iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/AbstractLimitsCheckTest.java

```diff
@@ -19,21 +19,88 @@
  */
 package org.sonar.iac.kubernetes.checks;
 
+import javax.annotation.Nullable;
 import org.junit.jupiter.api.Test;
 import org.sonar.iac.common.api.checks.CheckContext;
 import org.sonar.iac.common.api.tree.HasTextRange;
+import org.sonar.iac.common.api.tree.impl.TextRange;
+import org.sonar.iac.common.api.tree.impl.TextRanges;
+import org.sonar.iac.common.checks.Trilean;
+import org.sonar.iac.common.yaml.YamlParser;
 import org.sonar.iac.common.yaml.object.BlockObject;
+import org.sonar.iac.common.yaml.tree.MappingTree;
+import org.sonar.iac.kubernetes.visitors.KubernetesCheckContext;
+import org.sonar.iac.kubernetes.visitors.ProjectContext;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 import static org.sonar.iac.kubernetes.checks.AbstractLimitsCheck.getFirstChildElement;
 
 class AbstractLimitsCheckTest {
+  static MappingTree CONTAINER;
```

Comment:
1. I was confused when I saw this name for the first time. Maybe `CONTAINER_TREE` will be a better name?
2. It is used only in one method, so it may be better to move it to that method.